troposphere/views/web_shell.py

```diff
@@ -21,18 +22,25 @@ def _create_signature(secret, *parts):
 
 def web_shell(request):
     response = HttpResponse()
+    shell_user = request.user
+    if request.session:
+        logger.info("request.session: %s" % request.session.items())
```

Comment:
Normally I'd lean toward pulling this, but since it's `info` and we are likely to leverage it to investigate for at least a couple of releases - I'm 👍